pkg/controller/statusmanager/pod_status.go

```diff
@@ -290,6 +299,27 @@ func (status *StatusManager) setLastPodState(
 	})
 }
 
+func (status *StatusManager) CheckCrashLoopBackOffPods(dName types.NamespacedName, lb map[string]string, name string) []string {
```

Comment:
A few minor cleanups:
* rename `lb` to `selector`, `name` to `kind.
* Write a quick docblock, something like

```
CheckCrashLoopBackOffPods checks for pods in the label selector with with 
any containers in state CrashLoopBackoff. It returns a human-readable string 
for any pod in such a state.

dName should be the name of a DaemonSet or Deployment.
```

Reply:
ack, I'll fix these up